tests/view/crudeapi/test_comment.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import pytest
+import werkzeug
+from http import HTTPStatus
+from timeless.reservations.views import CommentView
+from timeless.reservations.views import Comment
+
+"""
+    Tests for CommentView
+    @todo #222:30min Finish CommentView implementation and CommentView tests. 
+     Methods for GET, POST and DELETE operation must be implemented, after the 
+     implementation remove the skip annotations from these tests.
+
+"""
+
+
+@pytest.mark.skip
+def test_get_found_comment():
+    commentview = CommentView()
+    expected = Comment(
+        id=1,
+        employee_id=2,
+        body="Comments for GET method",
+        date=datetime.datetime
+    )
+    result = commentview.get(commentview, comment_id=5)
```

Comment:
@paulodamaso see my previous comment. `.get` method doesn take args. All request params are stored in `import flask.request` which you can import everywhere, it doesnt come in args.

I suggest you just write mocked request to API and check response code / body, like it's made, for example, in https://github.com/timelesslounge/timelessis/blob/master/tests/test_roles.py

wdyt?

Reply:
@vryazanov I'll do it, thanks for the directions